src/components/Heading.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const StyledH1 = styled.h1`
```

Comment:
`StyledHeading`. `StyledH1` is still somewhat opaque. Better to change this from an `<h1 />` element to an `<h2 />` element since there should only be a single `<h1 />` element on a webpage for SEO, and this component seems to be included in each section.